simplex.py

```diff
@@ -276,7 +269,7 @@ def run(self, r_data=None):
                 data = calculate.main(self.args_ff)
                 con_ff.score = compare.compare_data(r_data, data)
                 opt.pretty_ff_results(con_ff)
-                if con_ff.score < self.new_ffs[-2].score:
```

Comment:
Nope, you're right, this change is correct. However, a few lines later doesn't match the paper. Making that change now.